src/BenchmarkDotNet/Loggers/ILogger.cs

```diff
@@ -2,6 +2,14 @@
 {
     public interface ILogger
     {
+        string Id { get; }
+        
+        /// <summary>
+        /// If there are several loggers with the same <see cref="Id"/>,
+        /// only logger with the highest priority will be used.
+        /// </summary>
+        int Priority { get; }
+        
```

Comment:
imho we should rather give a validation error in such cases rather than extending every logger with two fields

Reply:
I'm not sure that I got the idea. Could you explain it in detail?

Reply:
> "If there are several loggers with the same only logger with the highest priority will be used."

What I meant is that we should rather give a validation error when user configures two loggers of the same type.